ArcGISRuntimeSDKQt_CppSamples/Maps/ChangeBasemap/main.cpp

```diff
@@ -90,3 +90,27 @@ void setAPIKey(const QGuiApplication& app, QString apiKey)
 
   Esri::ArcGISRuntime::ArcGISRuntimeEnvironment::setApiKey(apiKey);
 }
+
+void setAPIKey(const QGuiApplication& app, QString apiKey)
```

Comment:
This method already exists, and this causes a compiler error when trying to build the sample